lib/conditionals.md

```diff
@@ -5,8 +5,15 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional is used to control the flow of a program once it is executed.  It does this by using 'if', 'else', and 'elseif' statements. 
+Flow in a program can also be controlled using the comparison method.  This method uses comparators such as: <, >, ==, <=, >=, and != to determine whether or not
+to execute a block of code.
+If the condition following an 'if' statement is true, then Ruby will run the block of code following it.  If the statement is false, then Ruby will jump to 
+the next line of code.  Often this is an 'else' statement.  We can add multiple conditions to the code using 'elseif'.  Like an 'if' statement, an 'elseif' statement
+will run the following block of code only if the condition is true.  Otherwise, it will jump to the next 'elseif' or 'else' statement in the code. 
+
```

Comment:
You haven't yet hit upon the control flow that I was referring to, which is fine :) Let me clarify my question to help you research/think in the right direction. (By the way, I think this Ruby Prompt itself should be revised, and I've noted it down to improve it for next time.)

If I wanted to control the flow of code so that I run a command only if a condition is met, then I would use `if`. Your answer explains as much. But what if I wanted to write code such that a command runs until a particular condition ceases to be true.

For example, if I'm trying to fully paint a canvas yellow, what would the Ruby code for that look like? Assume you have a Ruby command called `paint_yellow` that paints the next empty section on the canvas and another Ruby command called `canvas_is_full` that returns `true` once the canvas is fully painted.

You could write this to solve the challenge:

```ruby
paint_yellow
paint_yellow
paint_yellow
```

But that depends on you knowing that there are only three sections to paint. Or you might try this:

```ruby
# FYI - The exclamation mark on the below line means "not".
if !canvas_is_full
  paint_yellow
end
```

But that will only paint *one* section of the canvas. It doesn't repeat until the canvas is actually full.

So what can we use to control the flow of our program so it keeps on painting sections until the canvas is full?